internal/scaffold/cmd.go

```diff
@@ -217,8 +217,12 @@ func addMetrics(ctx context.Context, cfg *rest.Config, namespace string) {
 // serveCRMetrics gets the Operator/CustomResource GVKs and generates metrics based on those types.
 // It serves those metrics on "http://metricsHost:operatorMetricsPort".
 func serveCRMetrics(cfg *rest.Config) error {
-	// Below function returns filtered operator/CustomResource specific GVKs.
-	// For more control override the below GVK list with your own custom logic.
+	// Below function returns filtered operator/CustomResource specific GVKs. For more control override the below
```

Comment:
The function below returns filtered operator/CR specific GVKs.  For more control, override the GVK list below with your own custom logic.